reporter/tap.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+
+function reporter( rep ) {
+	if ( rep === "tap" ) {
+		tapReporter();
+	}
+};
+
+reporter.print = function( val ) {
+	console.log( val );
+};
+
+QUnit.reporter = reporter;
```

Comment:
This _definitely_ doesn't belong in reporter/tap.js, and I [still](https://github.com/jquery/qunit/issues/890#issuecomment-152739427) have issues with `QUnit.reporter( string )`.